features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
```

Comment:
You forgot to include the head. The first few lines should look in this document: like https://raw.githubusercontent.com/pabloNZ/openhab-docs/gh-pages/features/sitemap.md